javatests/com/williamfiset/algorithms/dp/MaximumSubarray.java

```diff
@@ -0,0 +1,21 @@
+package javatests.com.williamfiset.algorithms.dp;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.primitives.Ints;
+import com.williamfiset.algorithms.dp.MaximumSubarray;
+import com.williamfiset.algorithms.utils.TestUtils;
+import java.util.*;
+import org.junit.*;
+
+public class MaximumSubarray.java {
+
+  @Test
+  public void maximumSubarray() {
+    MaximumSubarray maxSS = new MaximumSubarray();
```

Comment:
You don't need to create an instance of a MaximumSubarray to call the method maximumSubarrayValue. You can simply call it because it is static:

int[] array = {1, 2, 1, -7, 2, -1, 40, 25};
int maxValue = MaximumSubarray.maximumSubarrayValue(array);